covid_xprize/validation/predictor_validation.py

```diff
@@ -68,12 +69,18 @@ def _check_columns(expected_columns, pred_df):
     missing_columns = expected_columns - set(pred_df.columns)
     if missing_columns:
         errors.append(f"Missing columns: {missing_columns}")
+        # Columns are not there, can't check anything more
         return errors
+
+    # Make sure column Date contains dates
+    date_column_type = pred_df[DATE].dtype
+    if not np.issubdtype(date_column_type, np.datetime64):
```

Comment:
That's the check. If Pandas didn't manage to parse the column as date, the column contains bad dates.